identity-service/src/routes/userBankTransactionMetadata.js

```diff
@@ -0,0 +1,94 @@
+const models = require("../models");
+const {
+  handleResponse,
+  successResponse,
+  errorResponseBadRequest,
+  errorResponseServerError,
+} = require("../apiHelpers");
+const { logger } = require("../logging");
+const { Op } = require("sequelize");
+
+const requiredPurchaseKeys = {
+  discriminator: { type: "string" },
+  usd: { type: "string" },
+  sol: { type: "string" },
+  audio: { type: "string" },
+  swapTransaction: { type: "string" },
+  buyTransaction: { type: "string" },
+};
+
+const validateMetadata = (metadata) => {
```

Comment:
I would support finding a light weight package to perform this type of validation for us, rather than hand rolling it - esp since this kind of validation isn't really tied to this particular method

Reply:
Yeah I don't feel great about this - do you have any recommendations on packages?

Reply:
maybe JSONSchema? we use this on DN

https://www.npmjs.com/package/jsonschema